WordPress/Sniffs/VIP/RestrictedFunctionsSniff.php

```diff
@@ -104,6 +104,64 @@ public function getGroups() {
 				),
 			),
 
+			'get_children' => array(
+				'type' => 'error',
+				'message' => '%s is prohibited, please use WP_Query instead.',
+				'functions' => array(
+					'get_children',
+				),
+			),
+
+			'wp_get_object_terms' => array(
+				'type' => 'error',
+				'message' => '%s is prohibited, please use get_the_terms() along with wp_list_pluck to extract the IDs.',
+				'functions' => array(
+					'wp_get_object_terms'
+				),
+			),
+			
+			'wp_get_post_(categories|tags|terms)' => array(
+				'type' => 'error',
+				'message' => '%s is prohibited, please use the equivalent get_the_* version instead (e.g. get_the_category())',
+				'functions' => array(
+					'wp_get_post_terms',
+					'wp_get_post_categories',
+					'wp_get_post_tags'
```

Comment:
Should add a trailing comma, and can also put into alphabetical order so they are the same order as the parent key.